charts/rancher-backup/Chart.yaml

```diff
@@ -1,3 +1,13 @@
+apiVersion: v2
+name: rancher-backup
+appVersion: 999
+version: 999
```

Comment:
Is appVeresion and version correct here and in the other Chart.yaml?

Reply:
Yes. This is changed in `scripts/package-helm`. This reflects rancher/operator as well: https://github.com/rancher/rancher-operator/blob/master/charts/rancher-operator/Chart.yaml

Reply:
Unresolving this comment since I'm not sure if I understand the context / agree with the idea that 999 should be used for the appVersion and version. @nickgerace can you elaborate on why this change is required?

From what I see in the scripts/package-helm script, it shouldn't matter if this is an actual version or if this is a 999, so I think we should leave the appVersion / version here and keep it up to date, even if the final chart will have its version "fixed" by the packaging script.

Reply:
@aiyengar2 I think setting it to a static value could be problematic if we miss it. It's not backed by CI. We would have to "manually" ensure that the versions are correct in this `Chart.yaml`.

Moreover, I think this chart will only be manually installed for development purposes, like `rancher/rancher-operator`. Having it follow that pattern would ensure that we don't "miss a spot" when updating the chart.

With that in mind my answer is a question: if we aren't using the version strings anyway since the operator is primarily installed via the Cluster Explorer UI, then what purpose do we have in tracking these versions?

If we determine that we do need to track these versions, then perhaps we change the CI to reflect that. Thoughts?

Reply:
Re-resolving this conversation after a conversation with @ibuildthecloud and @aiyengar2.